spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper do
+  let(:recipe) { Recipe.create!(name: name, description: description, user: user) }
+  let(:name) { 'existing recipe' }
+  let(:description) { 'existing recipe description' }
+
+  let(:current_user) { user }
+
+  let(:user) { User.create!(email: email, name: username, password: '1234567') }
+  let(:username) { 'Louise' }
+  let(:email) { 'test@test.com' }
+  let(:password) { 'louiseisverycool' }
+
+  before do
+    sign_in user if user
+  end
+
+  describe 'can_edit_recipe?' do
+    subject { helper.can_edit_recipe?(recipe) }
+    let(:recipe_owner) { true }
+    let(:admin?) { true }
```

Comment:
Could you set this to false, so that the first test is testing the "owner" part in the code.